core/imagedownloader.cpp

```diff
@@ -163,25 +177,30 @@ static QImage getThumbnailFromCache(const QString &picture_filename)
 			if (pictureTime.isValid() && thumbnailTime.isValid() && thumbnailTime < pictureTime) {
 				// Both files exist, have valid timestamps and thumbnail was calculated before picture.
 				// Return an empty thumbnail to signal recalculation of the thumbnail
-				return QImage();
+				return { QImage(), MEDIATYPE_UNKNOWN };
 			}
 		}
 	}
 
 	if (!file.open(QIODevice::ReadOnly))
-		return QImage();
+		return { QImage(), MEDIATYPE_UNKNOWN };
 	QDataStream stream(&file);
 
 	// Each thumbnail file is composed of a media-type and an image file.
-	// Currently, the type is ignored. This will be used to mark videos.
 	quint32 type;
 	QImage res;
 	stream >> type;
 	stream >> res;
-	return res;
+
+	// Thumbnails of videos currently not supported - replace by dummy
+	// TODO: Correctly extract thumbnails
```

Comment:
should we really keep this TODO?

Reply:
Maybe tone it down to
// TODO: Possibly extract thumbnail
?